records_mover/db/redshift/records_copy.py

```diff
@@ -13,18 +13,32 @@ def redshift_copy_options(unhandled_hints: Set[str],
                           fail_if_row_invalid: bool,
                           max_failure_rows: Optional[int]) -> RedshiftCopyOptions:
     redshift_options: RedshiftCopyOptions = {}
-    redshift_options['compression'] = hints['compression']
+    if hints['compression'] == 'GZIP':
+        redshift_options['compression'] = Compression.gzip
+    elif hints['compression'] == 'LZO':
+        redshift_options['compression'] = Compression.lzop
+    elif hints['compression'] == 'BZIP':
+        redshift_options['compression'] = Compression.bzip2
+    else:
+        cant_handle_hint(fail_if_cant_handle_hint, 'compression', hints)
+        redshift_options['compression'] = hints['compression']
```

Comment:
I noticed below for encoding you have a fallback to a fixed value in the else clause, but here you pass the value through directly. Is that intentional?

Reply:
Fair enough.  Come to think of it, consulting the Enum seems like a reasonable approach that would allow an old records mover to work with a new sqlalchemy-redshift, so I'll give that a shot.